src/jit/emitxarch.cpp

```diff
@@ -10791,6 +10791,10 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
             dst += emitOutputWord(dst, code);
             dst += emitOutputByte(dst, emitGetInsSC(id));
             sz = emitSizeOfInsDsc(id);
+
+            // Update GC info.
```

Comment:
You might want to have a little more comment here to answer the inevitable question when browsing the code, "why do you need to change the GC info of a shift destination?"